brainstormforce/ultimate-addons-for-gutenberg
# Fix: Block Conditions - Visible only to UAGB Block.

### Description

Trello - https://trello.com/c/2L8KlvMF/210-display-block-condition-only-to-uag-block

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop UAG block and check display condition is visible in all blocks or not.
- Drag and drop any block like core or any competitor block and our display condition should not visible in that block.
- Also check if Responsive visibility working properly for a block.

### Checklist:
